lib/qtermwidget.cpp

```diff
@@ -306,6 +306,8 @@ void QTermWidget::init(int startnow)
 
 QTermWidget::~QTermWidget()
 {
+    delete ColorSchemeManager::instance();
```

Comment:
Oops... thanks for reminding. I'll try to improve this within next couple of days.